### ISSUE TITLE
Feature request: clipcat-menu typing via xdotool

### ISSUE THREAD

**Issue by benedikt-bartscher** [Icelk/clipcat#18]
I find myself in many situations where i want to paste stuff from my clipboard to a VM or a RDP/VNC session where my hosts clipboard isnt available. Adding a mode to clipcat-menu which instead of only writing the selection in PRIMARY also types the selection via xdotool would be helpful.

**Comment by Icelk** [comment]
That's true. Do you think adding a config option to clipcatd (which is what does the actual pasting) which gives it a program to run when pasting? The data would be fed to stdin. And we'll probably make it so that pasting is ALSO done by the configured program, not ONLY by it.

**Comment by benedikt-bartscher** [comment]
Oh i am sorry i missunderstood the client-server architecture of clipcat. I somehow thought clipcatd was a server just storing the clipboard content and different computers (clients) connect to it.
But anyways, i like your solution of just specifing a additional command which gets fed by stdin. Any yes, the command shoud ALSO be executed.
